components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -811,7 +811,7 @@ impl Animate for Visibility {
 impl ComputeSquaredDistance for Visibility {
     #[inline]
     fn compute_squared_distance(&self, other: &Self) -> Result<SquaredDistance, ()> {
-        Ok(SquaredDistance::Value(if *self == *other { 0. } else { 1. }))
+        Ok(SquaredDistance::from_sqrt(if *self == *other { 0. } else { 1. }))
```

Comment:
Same, I'd really hope the compiler manages to inline all this and figure out that it doesn't actually need to do any multiplication. I'd say it's a safe bet but... :)